cli/cook/subcommands/tail.py

```diff
@@ -124,14 +126,20 @@ def tail(clusters, args, _):
     lines = args.get('lines')
     follow = args.get('follow')
     sleep_interval = args.get('sleep-interval')
+    wait_seconds = args.get('wait')
+
+    if wait_seconds == -1:
+        wait_seconds = None
+    elif not wait_seconds:
```

Comment:
In Python, `not 0.0` is `True`. I don't think that leads to the behavior we want here.